### DISCUSSION TITLE
E2 Interface Support for RIC

### DISCUSSION THREAD

**Discussion by suttonpd** [srsran/srsRAN_Project#163] [Announcements]
The srsRAN Project now supports 3rd party RIC frameworks. 
Our tutorial explains how the new E2 interface can be used: https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html
The Key Performance Measurement (KPM) Service Model is currently supported with RAN Control next on the roadmap. 

Happy Hacking!

**Comment by ryantheelder** [comment]
Awesome news! Looking forward to the future with RAN Control on the horizon. Keep up the great work! 👏🚀 #HappyHacking

**Reply by rolandmartinson7890** [reply]
Looking forward to the RAN Control SM. Well done srsRAN!

**Reply by natanzi** [reply]
Well done, please support Handover over E2.
